examples/bal/ex_bal.cpp

```diff
@@ -0,0 +1,55 @@
+#include "daisysp.h"
+#include "daisy_seed.h"
+
+using namespace daisysp;
+using namespace daisy;
+
+static DaisySeed seed;
+static Oscillator osc1, osc2;
+static Bal bal;
+
+static void AudioCallback(float *in, float *out, size_t size)
+{
+  float sig1, sig2;
+    for (size_t i = 0; i < size; i += 2)
+    {
+    	sig1 = osc1.Process();
+	sig2 = osc2.Process();
+	sig1 = bal.Process(sig1, sig2);
+	
+    	// left out
+        out[i] = sig1;
+
+        // right out
+        out[i+1] = sig1;
+    }
+}
```

Comment:
A more illustrative example might have the amplitude of sig2 changing so that the effect on sig1 can be heard.